### PR TITLE
FAIE-1674 support tiling for AD

### PR DESCRIPTION
**Pull request by nickjyj**

### CODE REVIEW

File: lmi_utils/image_utils/tiler.py [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+        """
+        obj = cls(0,0) # init an obj using dummy sizes
+        metadata = torch.load(pt_path)
+        for k,v in metadata.items():
+            setattr(obj,k,v)
+        return obj
+        
+    
+    @torch.inference_mode()
+    def tile(self, im:torch.Tensor, mode=ScaleMode.PADDING) -> torch.Tensor:
```
Comment (RugvedMavidipalli) [inline lmi_utils/image_utils/tiler.py:134]: should also add support for overlap ratio.

Comment (nickjyj) [reply lmi_utils/image_utils/tiler.py:134]: It supports overlap ratio. You can specify the stride paramater in the Tiler init function.

**Review by RugvedMavidipalli — APPROVED** [review]
Looks good to me

**Pull request merged**